include/roboptim/trajectory/visualization/cubic-b-spline-matplotlib.hh

```diff
@@ -49,6 +49,7 @@ namespace roboptim
           double nbp = static_cast<double> (spline.getNumberControlPoints());
           double intervals = nbp - 3;
           CubicBSpline::polynomials3vector_t polynomials;
+          CubicBSpline::knots_t kv = spline.knotVector();
```

Comment:
There is no need to take a copy here, as the spline won't be modified in the plotting operation.
For performance purposes (even thought talking about performance issues on a plotting function does not really make sense IMHO) and to avoid useless copies, I update the PR according to this remark